exercises/fizz-buzz/example.R

```diff
@@ -0,0 +1,16 @@
+fizz_buzz <- function(input) {
+  condition <- function(i) {
```

Comment:
[condition](https://en.wikipedia.org/wiki/Conditional_(computer_programming)) is quite a specific term. I'd be happies if we found a better name for it here, but I'm also OK with merging as-is. How about `replace_number`, because that's what the ReadMe suggests?

Reply:
happy to change  it to `replace_ number`